resourses/style.css

```diff
@@ -53,6 +50,11 @@ section:last-child{
   border: none;
 }
 
+section:first-child{ width: 100%; }
```

Comment:
Try to avoid adding specific selectors like first-child to generic tags as this will affect UI atleast in future modificaions.

Reply:
They are removed and styling is done with class names.